frontend/src/interfaces/INoteResult.ts

```diff
@@ -0,0 +1,6 @@
+export interface INoteResult {
```

Comment:
If this is a model that matches the backend it should be moved to the models/api/ and be named like the other Api models

Reply:
This looks like a duplicate of Api_Note. Are both necessary?

Reply:
Changes done, removed the INoteResult